hack/chains/create-signing-secret.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+# Show a message and exit gracefully if cosign isn't installed
+! which cosign >/dev/null 2>&1 && echo "Please install cosign!" && exit
```

Comment:
From where?

Reply:
Sure but first google result is https://www.cosign.be/ , enough to mention https://github.com/sigstore/cosign

Reply:
Since I'm making some new commits now, let me add a url.